app/views/support_interface/application_forms/_application_navigation.html.erb

```diff
@@ -1,4 +1,46 @@
+<% content_for :before_content do %>
+  <% if title == 'Details' %>
+    <%= render BreadcrumbComponent.new(items: [
+      {
+        text: 'Candidates',
+        path: support_interface_candidates_path
+      },
+      {
+        text: @application_form.candidate.email_address,
+        path: support_interface_candidate_path(@application_form.candidate)
+      },
+      {
+        text: @application_form.support_reference
+      }
+    ]) %>
+  <% else %>
+    <%= render BreadcrumbComponent.new(items: [
+      {
+        text: 'Candidates',
+        path: support_interface_candidates_path
+      },
+      {
+        text: @application_form.candidate.email_address,
+        path: support_interface_candidate_path(@application_form.candidate)
+      },
+      {
+        text: @application_form.support_reference,
+        path: support_interface_candidate_path(@application_form.candidate)
+      },
+      {
+        text: title
+      }
+    ]) %>
+  <% end %>
+<% end %>
```

Comment:
I’ve done this in a few places in this PR. When a page is the first ‘Details’ tab, we don’t to show that in the breadcrumb (only for subsequent tabs). Is there a smarter way of writing this - either in ERB template or perhaps somewhere in a controller/helper?